src/items/static-items.md

```diff
@@ -5,30 +5,23 @@
 > &nbsp;&nbsp; `static` `mut`<sup>?</sup> [IDENTIFIER] `:` [_Type_]
 >              `=` [_Expression_] `;`
 
-A *static item* is similar to a *constant*, except that it represents a precise
+A *static item* is similar to a [constant], except that it represents a precise
 memory location in the program. A static is never "inlined" at the usage site,
 and all references to it refer to the same memory location. Static items have
 the `static` lifetime, which outlives all other lifetimes in a Rust program.
-Static items may be placed in read-only memory if they do not contain any
-interior mutability.
+Static items may be placed in read-only memory if the type is not [interior 
+mutable]. Static items do not call `drop` at the end of the program.
```

Comment:
Have we ever used "interior mutable" as an adjective before? I always thought the term was always "interior mutability", as a noun.

Reply:
I don't think we do, but it is a natural analog to *interior mutability*.

Reply:
It's a shame "interiorally" isn't a word, otherwise that's what i would suggest. With our conversation on IRC, i'm willing to call this okay. Farther bikeshedding of how to decline "interior mutability" can occur elsewhere.